Entity/CategoryManager.php

```diff
@@ -143,11 +143,11 @@ public function getRootCategory($context = null)
     /**
      * NEXT_MAJOR: add this method to the interface.
      *
-     * @param ContextInterface $context
+     * @param ContextInterface|string|null $context
      *
      * @return CategoryInterface[]
      */
-    public function getRootCategoriesForContext(ContextInterface $context = null)
+    public function getRootCategoriesForContext($context = null)
```

Comment:
> because they were stubbed in tests...

How hard would it be to change from stubs to real objects? Or to introduce an interface you could mock?